### PR TITLE
Provide initial structure of the repo

### LINKED ISSUE #1: Merge latest Arma Additions into CNTO Additions
See PR https://github.com/CntoDev/cnto-additions/pull/41 for details.

### PR DESCRIPTION
**Pull request by freghar**
Add README, COPYING, gitignore, etc. and also port over existing CNTO modifications from `cnto-additions-old` (turns out to be just the editor subcategories, @Seb105 didn't want his littlebird hud included, see #1, and there seems to be nothing else based on [my diff](https://gist.github.com/freghar/d6b1bf4044cd8f1f560fce3569f92ab8) from a previous PR - anything else there is just an `arma-additions` modification, to be handled via a different process.

Please read through the README and see if it makes sense, thanks.

I've also chosen CC-BY-SA as the default license since it's at least one-way compatible with other GPLv3 projects, generally has pretty clear legal precedent, and states clearly what one can and cannot do, ... but it's really up to you if you want GPL or MIT or whatever instead.

### CODE REVIEW

File: mod.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name = "CNTO Additions";
+author = "Various Artists";
```
Comment (milivojm) [inline mod.cpp:2]: Ha!

**Comment by milivojm** [comment]
Does this resolve #1 completely?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    armake keygen [-f] <keyname>
+    armake sign [-f] [-s <signature>] <privatekey> <pbo>
+    armake paa2img [-f] <source> <target>
+    armake img2paa [-f] [-z] [-t <paatype>] <source> <target>
+
+Specifically, to build cnto-additions, call this for every folder (called ie.
+`something`) in `addons`:
+
+    armake build -p something something.pbo
+
```
Comment (milivojm) [inline README.md:55]: I'll add here later powershell script for armake. All good.

**Review by milivojm — APPROVED** [review]

**Comment by freghar** [comment]
> Does this resolve #1 completely?

Yes, sorry, forgot to mention that. The A3S repository maintainer will have to do a few extra steps when pulling `arma-additions` from Steam, steps which I've sent to Clarke earlier, but copying here:
```
1) Go into the addons folder
2) Remove ai.pbo
3) Unpack misc_tiny_configs.pbo using PBO Manager
4) Go into the misc_tiny_configs folder, remove the sight_alignment folder, go back one level
5) Remove (original) misc_tiny_configs.pbo, pack the misc_tiny_configs folder using PBO Manager, remove the folder
6) Go back one level, move force_video_settings from optionals to addons
7) Go to addons\force_video_settings, rename $PBOPREFIX$ to $PREFIX$
8) Go back one level (to addons)
9) Pack force_video_settings using PBO Manager, remove force_video_settings folder
```

**Comment by milivojm** [comment]
Ok, noted that on Wiki for time being. 

https://github.com/CntoDev/cnto-additions/wiki/Repacking-A3AA-for-CNTO

**Pull request merged**